### ISSUE TITLE
normalize_tuple() change.

### ISSUE THREAD

**Issue by keanucui** [keras-team/tf-keras#249]
normalize_tuple() changed its behavior since version r2.6 [source code](https://github.com/keras-team/keras/blob/r2.11/keras/utils/conv_utils.py). It is called by the tf.keras.layers.Cropping1D() function.
case with tf 1.15
```
import tensorflow as tf
import numpy as np
input_shape = (2, 3, 2)
x = np.arange(np.prod(input_shape)).reshape(input_shape)
y = tf.keras.layers.Cropping1D(cropping=(-10,0))(x)
````

![image](https://user-images.githubusercontent.com/14941351/221460372-35178405-183e-4e0e-8380-e9af0f996ed8.png)

it's fine. But with tf2.11
```
import tensorflow as tf
import numpy as np
input_shape = (2, 3, 2)
x = np.arange(np.prod(input_shape)).reshape(input_shape)
y = tf.keras.layers.Cropping1D(cropping=(-1,0))(x)
````
there is a error.
![image](https://user-images.githubusercontent.com/14941351/221460460-879e1d47-833e-42ce-ac36-3139b26f03e3.png)

so how to use Cropping1D() in tensorflow 2.11 consistent with the behavior in tensorflow 1.15. Or is there another way to do it in tensorflow 2.11? Thanks so much for response.
